src/python/grpcio/grpc/_links/service.py

```diff
@@ -177,7 +178,10 @@ def _on_service_acceptance_event(self, event, server):
     call = service_acceptance.call
     call.accept(self._completion_queue, call)
     try:
-      group, method = service_acceptance.method.split(b'/')[1:3]
+      service_method = service_acceptance.method
+      if sys.version_info[0] >= 3:
```

Comment:
Since we are using `six` throughout the rest of the library for Python 3 compatibility, please maintain uniformity by changing this to `if six.PY3:`.